caffeine/src/main/java/com/percolate/caffeine/DialogUtils.java

```diff
@@ -31,13 +35,17 @@ public static AlertDialog quickDialog(final Activity context, final String messa
         final SpannableString s = new SpannableString(message); //Make links clickable
         Linkify.addLinks(s, Linkify.ALL);
 
-        Builder builder = new AlertDialog.Builder(context);
-        builder.setMessage(s);
-        builder.setPositiveButton(android.R.string.ok, closeDialogListener());
-        AlertDialog dialog = builder.create();
-        dialog.show();
-
-        ((TextView) dialog.findViewById(android.R.id.message)).setMovementMethod(LinkMovementMethod.getInstance()); //Make links clickable
+        if(dialog != null && dialog.isShowing()){
```

Comment:
@brentwatson I think if Activity A calls `<quickDialog>` then Activity B is started, wouldn't the dialog then dismiss (possible Window Leak) then when the user calls `<quickDialog>` it will fail `<if(dialog != null && dialog.isShowing()>` and create a new dialog.